exercises/rectangles/RectanglesTest.fs

```diff
@@ -9,109 +9,69 @@ open Rectangles
 
 [<Fact>]
 let ``No rows`` () =
-    let input = []
-    rectangles input |> should equal 0
+    let strings = []
+    rectangles strings |> should equal 0
 
 [<Fact(Skip = "Remove to run test")>]
 let ``No columns`` () =
-    let input = [""]
-    rectangles input |> should equal 0
+    let strings = [""]
+    rectangles strings |> should equal 0
 
 [<Fact(Skip = "Remove to run test")>]
 let ``No rectangles`` () =
-    let input = [" "]
-    rectangles input |> should equal 0
+    let strings = [" "]
+    rectangles strings |> should equal 0
 
 [<Fact(Skip = "Remove to run test")>]
 let ``One rectangle`` () =
-    let input = 
-      [ "+-+"; 
-        "| |"; 
-        "+-+" ]
-    rectangles input |> should equal 1
+    let strings = ["+-+"; "| |"; "+-+"]
```

Comment:
@ErikSchierboom is there a reason you stopped outputting these on multiple lines? I think they were visually easier to understand before this change.

Reply:
I think this line needs to be updated:
https://github.com/ErikSchierboom/fsharp/blob/18ab904e83809cb315d6298918a001a988d6743b/generators/Generators.fs#L1035